### ISSUE TITLE
Not correctly generate redirects.csv

### ISSUE THREAD

**Issue by jan-vince** [adrenth/redirect#33]
I have updated OctoberCMS to version 380 and Redirect plugin stopped creating CSV file.

I work with SQLite.

Only first rule is created and Exception from Writer is raised.

*There is no exception handler in PublishManager, so no visual error is raised.*

Can anyone else confirm this?

Edit: I have found that cause of this is 'requirements' field sent from PublishManager to Writer as an array. 

I was able to bypass this error with json_encode 'requirements' field like this:

````
class PublishManager
...
public function publish() {
...
$writer = Writer::createFromPath($this->redirectsFile, 'w+');

  foreach($redirects->toArray() as $row){

    $row['requirements'] = json_encode($row['requirements']);

        $writer->insertOne($row);

  }
...
}
````

**Comment by adrenth** [comment]
Hi Jan,

Thanks for noticing this issue and providing a solution! Much appreciated. I have fixed this issue and will be released in version 2.0.10. If you are happy with my plugin/support, please let others know by leaving a review on octobercms.com.

**Comment by jan-vince** [comment]
Sure, I will :)

Thanks for fixing it!
